### ISSUE TITLE
TRaining of text encoder and changing learning rate....

### ISSUE THREAD

**Issue by 1blackbar** [TheLastBen/fast-stable-diffusion#182]

**Comment by TheLastBen** [comment]
did you experiment with learning_rate=1e-6  ?

**Comment by 1blackbar** [comment]
https://wandb.ai/psuraj/dreambooth/reports/Dreambooth-training-analysis--VmlldzoyNzk0NDc3

Im currently experimenting with 1e and 2e overall i think resylts are bvetter, in this repo i got random eye colours sometimes but with new code of his im getting accurate colours like on photos, its very new so  still testing but i can already tell it is better

**Comment by 1blackbar** [comment]
Ok tested Your repo with 2400 steps and 25 imgs, wayyyy to much overfitting with 2e-6 , i cant change a style, goes into painting but not really that artist, just typical painting style, i think maybe 4e-6 would be fine, ill try that.

**Comment by TheLastBen** [comment]
Thanks for the feedback, great help.

**Comment by 1blackbar** [comment]
so now using 5e-6  and removing --train_text_encoder should give me exact same results as 2 days ago or code was significantly changed ?

**Comment by TheLastBen** [comment]
@1blackbar you'll get the same results

**Comment by 1blackbar** [comment]
i went overboard with 57 imgs and 5e-6,  1800 steps ,trainig went berzerk and overfit with muddy results.

**Comment by 1blackbar** [comment]
Trained on shiv repo today, he changed the code and vae is merget into the ckpt so you dont need separate vae.pt file , trained on 4 images and 800 steps , it gives pretty good results very fast.
He reproduced dreambooth results with the doggie with his default settings 
`  --pretrained_model_name_or_path=$MODEL_NAME \
  --pretrained_vae_name_or_path="stabilityai/sd-vae-ft-mse" \
  --instance_data_dir=$INSTANCE_DIR \
  --class_data_dir=$CLASS_DIR \
  --output_dir=$OUTPUT_DIR \
  --with_prior_preservation --prior_loss_weight=1.0 \
  --instance_prompt="photo of sks {CLASS_NAME}" \
  --class_prompt="photo of a {CLASS_NAME}" \
  --seed=1337 \
  --resolution=512 \
  --train_batch_size=1 \
  --train_text_encoder \
  --mixed_precision="fp16" \
  --use_8bit_adam \
  --gradient_accumulation_steps=1 \
  --learning_rate=1e-6 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=50 \
  --sample_batch_size=4 \
  --max_train_steps=800`

![image](https://user-images.githubusercontent.com/101844463/197384288-b63c5e1c-a942-42ee-aac3-7382f1de68c6.png)

So learning rate its just better to keep it low to avoid overfitting, he has it at 1e-6 but like i wrote he has sample batch size at 4 thats why his results are quick with such slow rate but how to reproduce the same settings in this repo ?

**Comment by TheLastBen** [comment]
by default the sample batch size is 4, and as for the new vae, it is already merged in the download model cell

**Comment by TheLastBen** [comment]
![download (1](https://user-images.githubusercontent.com/110583491/197384923-6359d788-d72a-4975-a445-ad41bb1ef045.png)
![download (3)](https://user-images.githubusercontent.com/110583491/197384929-691264b7-2dc2-4c1c-9587-ed7f38c5df7a.png)
![download (4)](https://user-images.githubusercontent.com/110583491/197384932-9a8920eb-915d-4f3e-bb23-fe625e3483f0.png)
![download (5)](https://user-images.githubusercontent.com/110583491/197384939-114d6f40-54c4-4754-8893-0f3f24a0fabc.png)
